<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualRenderWidget</class>
 <widget class="QWidget" name="ManualRenderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>580</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Файлы</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Входной MKV файл:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="inputMkvPathEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseInputButton">
          <property name="text">
           <string>Обзор...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Выходной MP4 файл:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="outputMp4PathEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseOutputButton">
          <property name="text">
           <string>Обзор...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="title">
     <string>Параметры рендера</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Пресет кодировщика:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="renderPresetComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="hardsubCheckBox">
       <property name="text">
        <string>Встроить субтитры в видео (Hardsub)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="hardsubOptionsGroup">
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="internalSubsRadio">
          <property name="text">
           <string>Использовать дорожку из MKV:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="subtitleTrackComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="externalSubsRadio">
          <property name="text">
           <string>Использовать внешний .ass файл:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="externalSubsPathEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="browseExternalSubsButton">
            <property name="text">
             <string>Обзор...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="renderButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>НАЧАТЬ РЕНДЕР</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
